idea-plugin/src/main/java/com/palantir/gradle/jdks/GradleJdksProjectService.java

```diff
@@ -109,6 +151,21 @@ public void processTerminated(@NotNull ProcessEvent _event) {
             consoleView.get().attachToProcess(handler);
             ProcessTerminatedListener.attach(handler, project, "Gradle JDK setup finished with exit code $EXIT_CODE$");
             handler.waitFor();
+            if (handler.getExitCode() != 0) {
+                Notification notification = NotificationGroupManager.getInstance()
```

Comment:
if the process has failed for any reason, a pop-up will appear: 

<img width="1725" alt="Screenshot 2024-09-06 at 12 28 17 PM" src="https://github.com/user-attachments/assets/9bb314cb-566f-4e2d-b2e7-57a31deb37c4">

I could also focus the `Gradle JDK setup` window tool by default. wdyt ?

Reply:
addressed in: https://github.com/palantir/gradle-jdks/pull/405/commits/aceac47034cba90dacecf8fe3f41de79337b5331